### ISSUE TITLE
cannot connect to the multipass socket

### ISSUE THREAD

**Issue by jpmieville** [canonical/multipass#2980]
**Describe the bug**
After upgrading the kernel to version 6.1, multipass is no more working. If run kernel version 5.15, it work

**To Reproduce**
kernel 6.1.19-1-MANJARO
1. `multipass list`

```
list failed: cannot connect to the multipass socket
Please ensure multipassd is running and '/var/snap/multipass/common/multipass_socket' is accessible
```

2. `multipass shell primary`

```
launch failed: cannot connect to the multipass socket                           
Please ensure multipassd is running and '/var/snap/multipass/common/multipass_socket' is accessible
```

**Expected behavior**
What did you expect to happen?

**Logs**
see the file (journalctl --unit 'snap.multipass*' -S '2023-03-20' > multipass.log)

```
● snap.multipass.multipassd.service - Service for snap application multipass.multipassd
     Loaded: loaded (/etc/systemd/system/snap.multipass.multipassd.service; enabled; preset: disabled)
     Active: active (running) since Mon 2023-03-20 18:02:08 CET; 24s ago
   Main PID: 2644 (multipassd)
      Tasks: 3 (limit: 18978)
     Memory: 2.5M
        CPU: 170ms
     CGroup: /system.slice/snap.multipass.multipassd.service
             ├─2644 /snap/multipass/8465/bin/multipassd --verbosity debug --logger platform
             └─2691 /snap/multipass/8465/bin/multipassd --verbosity debug --logger platform

mar 20 18:02:08 manjaro4 systemd[1]: Started Service for snap application multipass.multipassd.
mar 20 18:02:08 manjaro4 multipassd[2644]: Using AppArmor support
mar 20 18:02:08 manjaro4 multipassd[2644]: Starting dnsmasq
mar 20 18:02:08 manjaro4 multipassd[2644]: Applied AppArmor policy: multipass.dnsmasq
mar 20 18:02:08 manjaro4 multipassd[2691]: QProcess: Destroyed while process ("dnsmasq") is still running.
mar 20 18:02:08 manjaro4 multipassd[2691]: [0] started: dnsmasq --keep-in-foreground --strict-order --bind-interfaces ->
```

**Additional info**
 - OS: Manjaro Linux / Kernel: 6.1.19-1-MANJARO arch: x86_64 bits: 64 compiler: gcc v: 12.2.1
- `multipass version 1.11.0`  
- `multipass info --all`
```
info failed: cannot connect to the multipass socket
Please ensure multipassd is running and '/var/snap/multipass/common/multipass_socket' is accessible
```
- `multipass get local.driver`
```
info failed: cannot connect to the multipass socket
Please ensure multipassd is running and '/var/snap/multipass/common/multipass_socket' is accessible
```

[multipass.log](https://github.com/canonical/multipass/files/11020874/multipass.log)

**Comment by townsend2010** [comment]
Hi @jpmieville!

Thank you for the bug report.  I'm marking this as a duplicate of #2533.

**Comment by jpmieville** [comment]
`snap refresh multipass --channel edge`

resolved my issue as mention on #2533
